add node.js information to openai integration (#15244)
DataDog/integrations-core: openai/README.md

DEBUG:ddtrace.contrib.openai._logging.py:sent 2 logs to 'http-intake.logs.datadoghq.com'
```

<!-- xxz tab xxx -->
<!-- xxx tab "Node.js" xxx -->

### Installation

1. Enable APM and StatsD in your Datadog Agent. For example, in Docker:

```shell
docker run -d
  --cgroupns host \
  --pid host \
  -v /var/run/docker.sock:/var/run/docker.sock:ro \
  -v /proc/:/host/proc/:ro \
  -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
  -e DD_API_KEY=<DATADOG_API_KEY> \
  -p 127.0.0.1:8126:8126/tcp \
  -p 127.0.0.1:8125:8125/udp \
  -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true \
  -e DD_APM_ENABLED=true \
  gcr.io/datadoghq/agent:latest
```

2. Install the Datadog APM Node.js library.

```shell
npm install dd-trace
```

3. Inject your OpenAI Node.js application with Datadog.

```shell
DD_TRACE_DEBUG=1 DD_TRACE_BEAUTIFUL_LOGS=1 DD_SERVICE="my-service" \
  DD_ENV="staging" DD_API_KEY=<DATADOG_API_KEY> \
  NODE_OPTIONS='-r dd-trace/init' node app.js
```

**Note**: If the Agent is using a non-default hostname or port, be sure to also set `DD_AGENT_HOST`, `DD_TRACE_AGENT_PORT`, or `DD_DOGSTATSD_PORT`.

> See the [APM Node.js OpenAI documentation][9] for more advanced usage.

Reviewer: ```suggestion
See the [APM Node.js OpenAI documentation][8] for more advanced usage.
```

### Configuration

> See the [APM Node.js library documentation][8] for all the available configuration options.

Reviewer: ```suggestion
See the [APM Node.js library documentation][9] for all the available configuration options.
```

> #### Log Prompt & Completion Sampling

Reviewer 2: ```suggestion
#### Log prompt and completion sampling
```

> 
> #### Log Prompt & Completion Sampling

Reviewer: ```suggestion
#### Log prompt and completion sampling
```

> To enable log prompt and completion sampling, set the `DD_OPENAI_LOGS_ENABLED=1` environment variable. By default, 10% of traced requests will emit logs containing the prompts and completions.

Reviewer 2: ```suggestion
To enable log prompt and completion sampling, set the `DD_OPENAI_LOGS_ENABLED=1` environment variable. By default, 10% of traced requests emit logs containing the prompts and completions.
```

> To enable log prompt and completion sampling, set the `DD_OPENAI_LOGS_ENABLED=1` environment variable. By default, 10% of traced requests will emit logs containing the prompts and completions.

Reviewer: ```suggestion
To enable log prompt and completion sampling, set the `DD_OPENAI_LOGS_ENABLED=1` environment variable. By default, 10% of traced requests emit logs containing the prompts and completions.
```

To adjust the log sample rate, see the [APM library documentation][3].

**Note**: Logs submission requires `DD_API_KEY` to be specified.

### Validation

> Validate that the APM Node.js library can communicate with your Agent by examining the debugging output from the application process. Within the section titled "Encoding payload" you should see an entry with a `name` field and a correlating value of `openai.request`. A truncated version of this output looks like this:

Reviewer 2: ```suggestion
Validate that the APM Node.js library can communicate with your Agent by examining the debugging output from the application process. Within the section titled "Encoding payload," you should see an entry with a `name` field and a correlating value of `openai.request`. A truncated version of the output looks like this:
```

> Validate that the APM Node.js library can communicate with your Agent by examining the debugging output from the application process. Within the section titled "Encoding payload" you should see an entry with a `name` field and a correlating value of `openai.request`. A truncated version of this output looks like this:

Reviewer: ```suggestion
Validate that the APM Node.js library can communicate with your Agent by examining the debugging output from the application process. Within the section titled "Encoding payload" you should see an entry with a `name` field and a correlating value of `openai.request`. See below for a truncated example of this output:
```
